app/policies/pull_policy.rb

```diff
@@ -5,15 +5,15 @@ def index?
   end
 
   def new?
-    true
+    @in_draw_period
```

Comment:
Shouldn't this be `user.is_admin || @in_draw_period`?

Reply:
yep